src/viam/app/data_client.py

```diff
@@ -566,6 +569,63 @@ async def _data_capture_upload(self, metadata: UploadMetadata, sensor_contents:
         response: DataCaptureUploadResponse = await self._data_sync_client.DataCaptureUpload(request, metadata=self._metadata)
         return response
 
+    async def streaming_data_capture_upload(
+        self,
+        data: bytes,
+        part_id: str,
+        file_ext: str,
+        component_type: Optional[str] = None,
+        component_name: Optional[str] = None,
+        method_name: Optional[str] = None,
+        method_parameters: Optional[Mapping[str, Any]] = None,
+        data_request_times: Optional[Tuple[datetime, datetime]] = None,
+        tags: Optional[List[str]] = None,
+    ) -> str:
+        """Uploads the metadata and contents of streaming binary data.
+
+        Args:
+            data (bytes): the data to be uploaded.
+            part_id (str): Part ID of the resource associated with the file.
+            file_ext (str): file extension type for the data. required for determining MIME type.
+            component_type (Optional[str]): Optional type of the component associated with the file (e.g., "movement_sensor").
+            component_name (Optional[str]): Optional name of the component associated with the file.
+            method_name (Optional[str]): Optional name of the method associated with the file.
+            method_parameters (Optional[str]): Optional dictionary of the method parameters. No longer in active use.
```

Comment:
What does "No longer in active use" mean? Should we not even have this parameter in the SDK if it's "deprecated"?

Reply:
Yeah, good question! The `no longer in active use` language is cribbed from how `method_parameters` is annotated elsewhere in the SDK. I don't remember the details precisely, but I remember conversations with the data team basically being "We don't use this now, but we should still allow it to be passed because it does update on the backend."

Reply:
Cool; so sort just short of deprecated. Seems fine to me, just wanted to call out.